src/index.js

```diff
@@ -121,4 +130,67 @@ function populateManagerDash() {
   availableRooms.innerText = getNumberAvailableRooms().length; 
   revenueToday.innerText = `$${getTodaysRevenue()}`; 
   roomOccupancy.innerText = `${getTodaysOccupancy()}%`;
-}
\ No newline at end of file
+}
+
+//Manager dash right side
+
+function findMatchingUser() {
+  const searchTerm = document.getElementById('search-bar').value;
```

Comment:
I know when I search for things on the internet, I don't typically think to be case sensitive because I know that the site will typically figure it out for me. Adding in something here that would change the input to capitalize first letter of first and last name could help make this more user friendly!

Reply:
@ErinUntermeyer Great thought, I will add that to my backlog to fix!